src/sas/sascalc/doc_regen/regenmodel.py

```diff
@@ -59,10 +59,14 @@
 
 from sas.sascalc.doc_regen.makedocumentation import MAIN_DOC_SRC, generate_html
 
-from typing import Dict, Any
+from typing import Any, Dict, Union
 from sasmodels.kernel import KernelModel
 from sasmodels.modelinfo import ModelInfo
 
+
+PATH_LIKE = Union[Path, str, os.PathLike[str]]
```

Comment:
This type now exists in three different places. I would suggeset importing it from `makedocumentation.py` so only one place needs to be updated if pathing changes in the future.